scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -119,7 +119,6 @@ task AssertIdenticalOutputs {
         set -o errexit
         set -o nounset
         set -o pipefail
-        set -o xtrace
```

Comment:
😿 

I'd be sad to see these `xtrace`s wiped out as they have saved my bacon many times. Happy to discuss this at (post) standup tomorrow.